pytissueoptics/scene/solids/solid.py

```diff
@@ -140,3 +146,9 @@ def _setInsideMaterial(self):
 
     def contains(self, *vertices: Vector) -> bool:
         raise NotImplementedError
+
+    def isStack(self) -> bool:
```

Comment:
annoying because is really just used to check it isn't a stack, but that's applicable for everything else, except for the stack.